### ISSUE TITLE
'useUser' cannot be used as a value because it was exported using 'export type'.ts(1362)

### ISSUE THREAD

**Issue by adonisv79** [auth0/nextjs-auth0#1041]
### Checklist

- [X] The issue can be reproduced in the [sample app](https://github.com/auth0-samples/auth0-nextjs-samples/tree/main/Sample-01) (or N/A).
- [X] I have looked into the [README](https://github.com/auth0/nextjs-auth0#readme) and have not found a suitable solution or answer.
- [X] I have looked into the [examples](https://github.com/auth0/nextjs-auth0/blob/main/EXAMPLES.md) and have not found a suitable solution or answer.
- [X] I have looked into the [API documentation](https://auth0.github.io/nextjs-auth0/) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0/nextjs-auth0/issues) and have not found a suitable solution or answer.
- [X] I have searched the [Auth0 Community](https://community.auth0.com/c/sdks/5) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Description

Hi, although the code continues to run, I get this warning in typescript that I am not sure if I should just ignore. It seems that useUser and the rest of the exports inside '@auth0/nextjs-auth0/client' uses "export type" and typescript is warning when these are used as values.

```
'useUser' cannot be used as a value because it was exported using 'export type'.ts(1362)
client.d.ts(1, 1): 'useUser' was exported here.
```

```
'UserProvider' cannot be used as a value because it was exported using 'export type'.ts(1362)
client.d.ts(1, 1): 'UserProvider' was exported here.
```

This is even visible using the basic example in the code.
```
import { useUser } from '@auth0/nextjs-auth0/client'
import Layout from '../components/Layout'

export default AboutPage = () => {
  // warning appears here
  const { user, isLoading } = useUser()
  
  return (
    <Layout user={user} isLoading={isLoading}>
      <h1>About</h1>
    </Layout>
  )
}

```

### Reproduction

1. Using a basic example, and having the following versions 
    "@auth0/nextjs-auth0": "^2.2.0",
    "eslint": "8.32.0",
    "eslint-config-next": "13.1.5",
    "next": "13.1.5",
    "react": "18.2.0",
    "react-dom": "18.2.0",
    "typescript": "4.9.4"

### SDK version

^2.2.0

### Next.js version

13.1.5

### Node.js version

16.16.0

**Comment by ewanharris** [comment]
Hi @adonisv79, this issue was fixed in #1037 and released as `v2.2.1`, updating to that version should fix the issue

**Comment by adonisv79** [comment]
Fix verified. thanks
